RevenueCatUI/Templates/V2/ViewHelpers/BadgeModifier.swift

```diff
@@ -34,6 +34,7 @@ struct BadgeModifier: ViewModifier {
 
         var backgroundStyle: BackgroundStyle? {
             stack.backgroundColor?.asDisplayable(uiConfigProvider: uiConfigProvider).backgroundStyle
+                ?? stack.background?.asDisplayable(uiConfigProvider: uiConfigProvider).backgroundStyle
```

Comment:
I believe this should be the other way around, first use the background property if it exists, if not, fallback to the backgroundColor

Reply:
you're absolutely right!